src/main/java/net/sf/jabref/gui/search/SearchResultFrame.java

```diff
@@ -460,36 +538,40 @@ public Object getColumnValue(BibEntry entry, int column) {
                     } else {
                         return null;
                     }
-                case URL_COL:
-                    if (entry.hasField(FieldName.URL)) {
-                        urlLabel.setToolTipText(entry.getFieldOptional(FieldName.URL).get());
+                case URL_COL: {
+                    Optional<String> urlField = entry.getFieldOptional(FieldName.URL);
```

Comment:
Simpler: e`ntry.getFieldOptional(FieldName.URL).ifPresent(urlLabel::setToopTipText)`
The latter one is a nice way of writing method references in lambdas, see here for an overview:
https://blog.idrsolutions.com/2015/02/java-8-method-references-explained-5-minutes/